packages/pds/src/services/repo/index.ts

```diff
@@ -220,6 +225,10 @@ export class RepoService {
     commitData: CommitData,
     writes: PreparedWrite[],
   ) {
+    this.db.onCommit(() => {
+      this.crawlers.notifyOfUpdate()
```

Comment:
Looks like `notifyOfUpdate()` is designed not to throw, but if I didn't know that I'd probably recommend adding a `catch` here to avoid an unhandled rejection. We could add it anyway if we're feeling defensive.  I suppose I'm not too worried about it either way!

Reply:
yup I'll add it in just for peace of mind 👍